state_synchronizer/src/executor_proxy.rs

```diff
@@ -116,4 +125,12 @@ impl ExecutorProxyTrait for ExecutorProxy {
         }
             .boxed()
     }
+
+    fn validate_ledger_info(
+        &self,
+        target: &LedgerInfoWithSignatures<Ed25519Signature>,
+    ) -> Result<()> {
+        target.verify(&self.validator_verifier)?;
```

Comment:
nit: can you just do `target.verify(&self.validator_verifier)?` here without extra statements?

Reply:
Says mismatched type if i do `target.verify(&self.validator_verifier)?` and no Ok(()) return at the end